### ISSUE TITLE
Feature request: changeStop event

### ISSUE THREAD

**Issue by jaankoppe** [Simonwep/pickr#128]
#### Do you want to request a **feature** or report a **bug**? 
Feature
#### What is the current behavior?
Currently the 'change' event will fire on every change, for example when changing color by sliding.
#### What is the expected behavior?
It would be great when the changeStop event will fire only when the color change is stopped (e.g. mouseup)
This feature will come handy when trying to implement undo/redo functionality and saving state only on color changeStop.

PS! Great library - was looking for a vanilla javascript version of react-color and this looks like the best option so far!
#### Your environment:
```
Version (see Pickr.version): 3.2.0
Used bundle (es5 or normal one): normal
Used theme (default is classic): nano
Browser-version:  Chrome 75
Operating-system:  Windows 10
```

**Comment by Simonwep** [comment]
Added in bc3c4e657b2daaa929cfe307f019a424c4b0dae4 :) Available in `1.2.4` (where do you got `3.2.0` ?)

**Comment by jaankoppe** [comment]
> Added in [bc3c4e6](https://github.com/Simonwep/pickr/commit/bc3c4e657b2daaa929cfe307f019a424c4b0dae4) :) Available in `1.2.4` (where do you got `3.2.0` ?)

Thanks!
Sorry about the version, accidentally added Fabric.js version due to working with the both libraries on a same project :)

**Comment by Simonwep** [comment]
Oh haha, no problem - glad I could help :)
